.github/workflows/maintainance_bot.yaml

```diff
@@ -0,0 +1,11 @@
+name: "Maintenance Bot"
+on:
+- pull_request_target
+
+jobs:
+  triage:
+    runs-on: ubuntu-18.04
```

Comment:
```suggestion
    runs-on: ubuntu-latest
```

Reply:
why latest? I thought, using  persistent OS label is preferred. Because if new OS release breaks something, we won't be affected

Reply:
I guess I prefer to be aware if something doesn't work with a new release. Then we can decide to fix the issue or pin the release if/when it breaks.